bastion.tf

```diff
@@ -0,0 +1,38 @@
+# Create the Azure Bastion subnet and resource
+# Define the Azure Bastion subnet
+resource "azurerm_subnet" "bastion_subnet" {
+  count                = var.install_azure_bastion == true ? 1 : 0
+  name                 = "AzureBastionSubnet"
+  resource_group_name  = var.hub_resource_group_name
+  virtual_network_name = module.data_landing_zone.hub_vnet_name
+  address_prefixes     = var.services_bastion_subnet_address_space
+}
```

Comment:
Can you create the subnet via the networking module please? https://github.com/hmcts/terraform-module-data-landing-zone/blob/main/networking.tf